### DISCUSSION TITLE
kpanda/best-practice/add-worker-node-on-global/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3111] [Announcements]
# kpanda/best-practice/add-worker-node-on-global/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/best-practice/add-worker-node-on-global/

**Comment by gssjl2008** [comment]
runc 地址不一样，
 sed -i 's#/usr/local/bin/runc#/usr/local/sbin/runc#g' /etc/containerd/config.toml
容器内替换下，然后restart containerd

**Reply by gssjl2008** [reply]
不支持通过密钥安装的 global 集群， 需要手动通过 cr 进行创建节点，

**Comment by peterchenhc** [comment]
# 《为全局服务集群添加节点¶ 》 这个章节第三步需要注意：

**自定义参数**的添加是在 **check node**步骤之后执行

**Reply by windsonsea** [reply]
在 PR #5294 中更新了 check node 步骤的说明

**Comment by panpan0000** [comment]
这种页面，是否应该放入接入集群的文档的子页面，而不是无脑堆在“最佳实践”里？
